tidb-cloud/serverless-driver.md

```diff
@@ -1,12 +1,18 @@
 ---
 title: TiDB Cloud Serverless Driver (Beta)
-summary: Learn how to connect to TiDB Serverless from serverless and edge environments over HTTP.
+summary: Learn how to connect to TiDB Serverless from serverless and edge environments.
 aliases: ['/tidbcloud/serverless-driver-config']
 ---
 
-# TiDB Cloud Serverless Driver (Beta)
+# Why use TiDB Cloud Serverless Driver (Beta)
```

Comment:
The new title doesn't fully summarize the content of the entire document. Suggest using the previous one.

Reply:
It doesn't try to summarize the entire doc. It's just a separate title. Like this https://planetscale.com/docs/tutorials/planetscale-serverless-driver.

Reply:
Got it. Then you should add a L2 heading for the separate block.